lib/probes_.h

```diff
@@ -98,9 +104,22 @@ static inline void h2o_probe_log_request(h2o_req_t *req, uint64_t req_index)
     }
 }
 
-static inline void h2o_probe_log_response_status(h2o_req_t *req, uint64_t req_index)
+static inline void h2o_probe_log_response(h2o_req_t *req, uint64_t req_index)
 {
     H2O_PROBE_CONN(SEND_RESPONSE_STATUS, req->conn, req_index, req->res.status);
+    if (H2O_CONN_IS_PROBED(SEND_RESPONSE_HEADER, req->conn)) {
+        if (req->res.content_length != SIZE_MAX) {
+            h2o_iovec_t cl_buf;
+            cl_buf.base = h2o_mem_alloc_pool(&req->pool, char, sizeof(H2O_UINT64_LONGEST_STR) - 1);
+            cl_buf.len = sprintf(cl_buf.base, "%zu", req->res.content_length);
```

Comment:
I think we can use a local array here; i.e. `char clbuf[sizeof(H2O_UINT64_LONGEST_STR)]`.

Also, assuming that we use `H2O_UINT64_LONGEST_STR`, the format specifier should be `"%" PRIu64`.

Reply:
Thank you, that makes sense. Does 7c22c606 look more in line with your feedback? I retained `h2o_iovec_t` in order to keep the `h2o_probe_response_header` API consistent with other probe functions. What it points to internally, as you can see, is the local array.

Reply:
Thank you for the changes.

I do not think it's necessary to have two ways of referring to the same object (i.e., `buf` and `cl_buf.base`). Something like below uses less state and therefore easier to read:

```c
char buf[sizeof(H2O_UINT64_LONGEST_STR)];
size_t len = sprintf(buf, "%" PRIu64, req->res.content_length);
h2o_probe_response_header(req, req_index, H2O_TOKEN_CONTENT_LENGTH->buf, h2o_iovec_init(buf, len));
```

Reply:
Ah! `h2o_iovec_init` had completely slipped off my mind. Thank you for the refresher.